### ISSUE TITLE
Plan always showing changes unless empty RKE config elements defined

### ISSUE THREAD

**Issue by davidarcher** [rancher/terraform-provider-rancher2#19]
When defining a cluster using Rancher-launched Kubernetes on Openstack, my terraform plan always shows changes to the openstack_cloud_provider sub-objects that I haven't explicitly defined in my TF code.

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ rancher2_cluster.cluster
      rke_config.0.cloud_provider.0.openstack_cloud_provider.0.block_storage.#: "1" => "0"
      rke_config.0.cloud_provider.0.openstack_cloud_provider.0.metadata.#:      "1" => "0"
      rke_config.0.cloud_provider.0.openstack_cloud_provider.0.route.#:         "1" => "0"
      rke_config.0.network.0.weave_network_provider.#:                          "1" => "0"
```

To workaround this, I have to add empty objects to the TF code but this should really be handled by the provider.

```
resource "rancher2_cluster" "cluster" {
  name        = "${var.cluster_name}"
  description = "${var.cluster_description}"

  rke_config {
    cloud_provider {
      openstack_cloud_provider {
        global {
          ...
       }

        load_balancer {
          ...
        }

        #### WORKAROUND
        block_storage {}
        metadata {}
        route {}
      }
    }

    network {
      plugin = "weave"
      #### WORKAROUND
      weave_network_provider {
        password= ""
      }
    }
  }
}
```

**Comment by rawmind0** [comment]
@davidarcher, i could reproduce this behaviour if `block_storage`, `metadata`, `route ` and/or `load_balancer` are not specified in `cloud_provider` - `openstack_cloud_provider`, but not in the `network` section, this is working fine for me...

```
    network {
      plugin = "weave"
    }
```

`cloud_provider` section is fixed on PR #8

**Comment by rawmind0** [comment]
Fix for this issue is included on new release [v1.1.0](https://github.com/terraform-providers/terraform-provider-rancher2/releases/tag/v1.1.0) 

@davidarcher please, reopen issue if needed.
